### DISCUSSION TITLE
Hajk docker deployment

### DISCUSSION THREAD

**Discussion by matself** [hajkmap/Hajk#1471] [General]
docker-hajk README (from two years back) says 
**A containerized Hajk solution with everything you need supplied. Just run _docker compose up_ and you're good to go.**

Well it seems to be more than that.
I have Docker running and tested. I have a separate Nginx running and tested.
`Docker inspect hajk` shows no errors. I have docker-composer as well. Running the Hajk container installs a second Nginx container.
So all I need to do is to run the Hajk docker image so that it connects to either of the nginx webservers.
The only docker-compose.yml I find is in the docker subdirectory /diff/usr/app/node_modules/ldapjs/docker-compose.yml
I realize that I need to configure Nginx and/or Hajk to speak with each other.
And this is where my skills end. 
All this is running in an Ubuntu 22.04 machine.

**Comment by matself** [comment]
Jag stänger den här frågan - gav upp Docker-lösningen och fick med hjälp av @jesade-vbg igång en nodejs-instans istället. Plus långa konversationer med ChatGPT.
Installationsanvisningen till Docker-paketeringen måste ses över och verifieras. Två sidor hänvisar fram och tillbaks till varandra.

**Comment by jacobwod** [comment]
Let's refer to this thread as it contains the correct solution on what was missing in your initial configuration. Perhaps someone else will find it useful too. Swedish only. 

But basically, ensure to have the correct `mapServiceBase` in your `appConfig.json`.

https://github.com/hajkmap/Hajk/discussions/1473#discussioncomment-8431120
